docs/api.md

```diff
@@ -1,5 +1,14 @@
-## Properties: auro-popover
+# auro-popover
 
-| Attribute | Value type | Description |
-|----|----|----|
-| attr | string, boolean, function | Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. |
+## Properties
+
+| Property    | Attribute   | Type     |
+|-------------|-------------|----------|
+| `for`       | `for`       | `string` |
```

Comment:
There are no descriptions? See `Describe attributes of your component` at https://auro.alaskaair.com/getting-started/developers/generator/getting-started to add these descriptions.